### PR TITLE
Fix: Overlord Gatling Cannon Has Very Slow Firing Animation

### PR DESCRIPTION
**Pull request by commy2**
### 1.04

- Gatling on Overlord turns slowly when firing

### patch

- Gatling on Overlord turns exactly as fast as base defense Gatling:

https://user-images.githubusercontent.com/6576312/194762649-2570885b-ddc2-46ed-88cf-a6ce1987212b.mp4

- the values are calculated from model data

Notes:

- Overlord Gatling animation is shorter (thus faster) than base defense animation
- the Base Defense Gatling animation takes 75 frames to complete 1 cycle
- the Base Defense Gatling animation is scaled with these factors:

mode | animation speed factor
--|--
slow | 0.8
mid | 1.5
fast | 3.0

- the Overlord Gatling animation takes 30 frames to complete 1 cycle
- the Overlord Gatling animation is scaled with these factors:

mode | before | after
--|--|--
slow | 0.1 | 0.3
mid | 0.2 | 0.6
fast | 0.3 | 1.2

- the new values were calculated thus:

```py
0.75*/(75/30)  = 0.3
1.5  /(75/30)  = 0.6
3.0  /(75/30)  = 1.2

* 0.8 is presumably a result from rounding 0.75, so 0.75 was taken as base value
```

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/INI/Object/NukeGeneral.ini [hunk 1]
```diff
@@ -17403,9 +17403,9 @@ Object Nuke_ChinaTankOverlordGattlingCannon
   ;UpgradeCameo4 = NONE
   UpgradeCameo5 = Upgrade_ChinaOverlordGattlingCannon
 
-
   ; Patch104p @bugfix commy2 08/10/2022 Fix muzzle flash when aiming at grounded targets.
   ; Patch104p @bugfix commy2 08/10/2022 Fix cannon keeps spinning after firing once.
+  ; Patch104p @tweak commy2 09/11/2022 Tweak firing rotation animation speed.
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/NukeGeneral.ini]: I think this can be labeled as bugfix. It should not be that slow.

**Review by xezon — APPROVED** [review]
Looks good to me.

File: Patch104pZH/GameFilesEdited/Data/INI/Object/NukeGeneral.ini (hunk 1 above)
Comment (commy2) [reply Patch104pZH/GameFilesEdited/Data/INI/Object/NukeGeneral.ini]: Also, wrong date.
```suggestion
  ; Patch104p @bugfix commy2 09/10/2022 Fix slow rotation in firing animation.
```

File: Patch104pZH/GameFilesEdited/Data/INI/Object/TankGeneral.ini [hunk 2]
```diff
@@ -3183,9 +3183,9 @@ Object Tank_ChinaTankOverlordGattlingCannon
   ;UpgradeCameo4 = NONE
   UpgradeCameo5 = Upgrade_ChinaOverlordGattlingCannon
 
-
   ; Patch104p @bugfix commy2 08/10/2022 Fix muzzle flash when aiming at grounded targets.
   ; Patch104p @bugfix commy2 08/10/2022 Fix cannon keeps spinning after firing once.
+  ; Patch104p @tweak commy2 09/11/2022 Tweak firing rotation animation speed.
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/TankGeneral.ini]:
```suggestion
  ; Patch104p @bugfix commy2 09/10/2022 Fix slow rotation in firing animation.
```

File: Patch104pZH/GameFilesEdited/Data/INI/Object/ChinaVehicle.ini [hunk 3]
```diff
@@ -458,9 +458,9 @@ Object ChinaTankOverlordGattlingCannon
   ;UpgradeCameo4 = NONE
   UpgradeCameo5 = Upgrade_ChinaOverlordGattlingCannon
 
-
   ; Patch104p @bugfix commy2 08/10/2022 Fix muzzle flash when aiming at grounded targets.
   ; Patch104p @bugfix commy2 08/10/2022 Fix cannon keeps spinning after firing once.
+  ; Patch104p @tweak commy2 09/11/2022 Tweak firing rotation animation speed.
```
Comment (commy2) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/ChinaVehicle.ini]:
```suggestion
  ; Patch104p @bugfix commy2 09/10/2022 Fix slow rotation in firing animation.
```

**Pull request merged**